ebean-core/src/main/java/io/ebeaninternal/server/deploy/BeanProperty.java

```diff
@@ -1416,15 +1416,20 @@ private void jsonWriteScalar(SpiJsonWriter writeJson, Object value) throws IOExc
   }
 
   public void jsonRead(SpiJsonReader ctx, EntityBean bean) throws IOException {
+    Object objValue = jsonRead(ctx);
+    if (jsonDeserialize) {
+      if (ctx.update()) {
+        setValueIntercept(bean, objValue);
+      } else {
+        setValue(bean, objValue);
+      }
+    }
+  }
+
+  public Object jsonRead(SpiJsonReader ctx) throws IOException {
```

Comment:
I've split the method to support reading a property without setting it to the bean